docs/testrunner-toolkit/bundling.md

```diff
@@ -0,0 +1,103 @@
+---
+id: bundling
+title: Bundling Test Files
+sidebar_label: CLI Reference
+---
+
+The [saucectl command line interface](saucectl) runs tests (Cypress, TestCafe, Playwright, ...) by bundling up your root directory, sending the bundle to the Sauce Labs cloud (or in your own infrastructure via Docker), unpacking the bundle and then running the tests. This page explains how bundling works and how you can reduce the size of your bundle for improved performance.
+
+## Bundling
+
+The Sauce Labs config.yml has a parameter called "rootDir" that tells saucectl where your test files are ([see configuration documentation](configuration.md)). These test files get bundled and uploaded to the Sauce Labs cloud.
+
+![img](../../static/img/saucectl/uploading-project.png)
```

Comment:
You'll need to add this line:

`import useBaseUrl from '@docusaurus/useBaseUrl';`

right below the front matter and then replace the markdown for the image ref tag to this:

`<img src={useBaseUrl('img/saucectl/uploading-project.png')} alt="Uploading a Project" />`

Otherwise we get broken image tags once CI runs `npm run build`